moto_week3/s3_func_test.py

```diff
@@ -0,0 +1,72 @@
+import boto3
+import io
+import sys
+import unittest
+from moto import mock_s3
+from s3_functions import main, get_client, list_s3_buckets, list_s3_objects, read_s3_object
+
+
+class S3Tests(unittest.TestCase):
+    def setUp(self):
+        #sets up the local moto s3 service for mocking.
+        self.bucket = 'buck02jay'
+        self.key = 'upload.txt'
+        self.value = 'RA1511001010001'
+
+    @mock_s3
+    def __moto_setup(self):
+        
+        """
+        simulation of s3 file upload
+        """
+        s3 = get_client()
+        s3.create_bucket(Bucket=self.bucket)
+        s3.put_object(Bucket=self.bucket, Key=self.key, Body = self.value)
+        pass
+
+    @mock_s3
+    def test_get_client(self):
+        s3 = get_client()
+        #success
+        self.assertEqual(s3._endpoint.host, "https://s3.ap-south-1.amazonaws.com")

```

Comment:
Revisit this to see if boto3.client returns s3 endpoint with or without region.

Reply:
on it.